packages/ckeditor5-heading/docs/features/headings.md

```diff
@@ -26,6 +26,17 @@ By default, when your editor build does not include the title plugin, a `<h1>` e
 	You can read more about why the editor should not create `<h1>` elements for content headings in the [Headings section of Editor Recommendations](http://ckeditor.github.io/editor-recommendations/features/headings.html).
 </info-box>
 
+## Demo
```

Comment:
Something's wrong here with the demos and structure, both before your changes and after them. I'd see the structure like this:

# Title

Feature description

## Heading levels

## Demo

(Demo with default setup)

## Related features

List

## Configuration

### Configuring heading levels

With demo

### Configuring custom heading elements

With demo

## Installation

## Common API

## Contribute